polaris-react/src/components/VideoThumbnail/README.md

```diff
@@ -33,7 +33,7 @@ Video thumbnails should:
 
 ## Examples
 
-### Basic video thumbnail
+### Default
```

Comment:
Help a tired person out: Why do we not delete these readmes? Aren't they 1:1 with what's in the website directory?

Reply:
I literally did this so I could make sure we didn't lose any content as it hadn't been migrated across in a long time. We definitely should delete them but first I need to make sure the content is not lost.

Reply:
Gotcha!